test/commands/console/index.test.js

```diff
@@ -82,8 +87,11 @@ describe('ConsoleCommand', () => {
           return null
         })
         command.printConsoleConfig()
-        expect(command.log).toHaveBeenCalledTimes(1)
-        expect(command.log).toHaveBeenCalledWith(noOrgErrorMessage)
+        expect(command.log).toHaveBeenCalledTimes(4)
+        expect(command.log).toHaveBeenCalledWith('You are currently in:')
+        expect(command.log).toHaveBeenCalledWith('1. Org: <no org selected>')
+        expect(command.log).toHaveBeenCalledWith('2. Project: <no project selected>')
+        expect(command.log).toHaveBeenCalledWith('3. Workspace: THE_WORKSPACE')
```

Comment:
same here. for a workspace to be selected, project and org needs to be selected. I get that this tests a permutation, but the config shouldn't ever get to this state (can the test be removed instead?)